tests/Unit/Schema/Directives/Fields/CanDirectiveTest.php

```diff
@@ -5,26 +5,31 @@
 use Tests\TestCase;
 use Tests\Utils\Models\User;
 use Nuwave\Lighthouse\Exceptions\AuthorizationException;
-use Nuwave\Lighthouse\Exceptions\AuthenticationException;
 
 class CanDirectiveTest extends TestCase
 {
     /**
      * @test
+     * @dataProvider provideAcceptableArgumentNames
+     *
+     * @param string $argumentName
      */
-    public function itThrowsWhenNotAuthenticated()
+    public function itThrowsIfNotAuthorized(string $argumentName)
     {
-        $schema = '
+        $this->be(new User());
+
+        $schema = sprintf('
```

Comment:
String concatenation is preferred for consistency

Reply:
I thought this is much more cleaner than concatenation…
Also, PHPStorm shows us where the variables are used.

![phpstorm](https://preview.ibb.co/bGSejf/Screen-Shot-2018-11-20-at-19-22-00.png)

What do you think?

If you really want me to rollback it to the concatenation style, I'll do it later on 🖐🏼

Reply:
I prefer inlining and most of all, i prefer consistency.

Reply:
OK! I just changed them to string concatenation 😃